x/ibc/07-tendermint/client/cli/tx.go

```diff
@@ -170,3 +184,25 @@ $ %s tx ibc client misbehaviour [path/to/evidence.json] --from node0 --home ../n
 
 	return cmd
 }
+
+func parseFraction(fraction string) (tmmath.Fraction, error) {
```

Comment:
cc: @marbar3778 @melekes maybe we should upstream this function to tendermint

Reply:
good idea